tests/test_measure.py

```diff
@@ -351,7 +351,7 @@ def testPsfFlux(self):
         schema.addField("centroid_x", type=float)
         schema.addField("centroid_y", type=float)
         schema.addField("centroid_flag", type='Flag')
-        sfm_config = measBase.SingleFrameMeasurementConfig()
+        sfm_config = measBase.SingleFrameMeasurementConfig(ignoreSlotPluginChecks=True)
```

Comment:
This will need to be wrapped to ignore/check warnings if the meas_base change is done.